packages/jest-runner/src/abq.ts

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {loadJson} from 'json.macro';
+
+const abq = <any>loadJson('../../../abq.json');
+const lerna = <any>loadJson('../../../lerna.json');
```

Comment:
This is kludgy, but we want to separately track the underlying version of Jest we're wrapping (`testFrameworkVersion`) and our own version (`adapterVersion`).

This is a Babel macro that inlines the file into the output, since `abq.json` and `lerna.json` aren't part of the build child packages.